wiki/Contests/o!tFAC/3/en.md

```diff
@@ -23,38 +23,51 @@ The **osu!taiko Featured Artist Cup #3** (***o!tfac #3***) is an osu!taiko mappi
 | Mapping phase | 2022-10-23/2022-11-06 00:00 |
 | Screening phase | 2022-11-07/2022-11-21 00:00 |
 | Judging phase | 2022-11-22/2022-12-06 00:00 |
-| Results | 2022-12-13 |
+| Results | 2022-12-23 |
 
 ## Organisation
 
-The osu!taiko Featured Artist Cup #3 is run by various community members.
+The osu!taiko Featured Artist Cup #3 was run by various community members.
 
 | Position | Member(s) |
 | :-- | :-- |
 | Host | ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) |
 | Designer | ::{ flag=GB }:: [Tanza3D](https://osu.ppy.sh/users/10379965) |
 | Judge | ::{ flag=AR }:: [Axer](https://osu.ppy.sh/users/7299864), ::{ flag=NL }:: [Boaz](https://osu.ppy.sh/users/13302996), ::{ flag=US }:: [Nifty](https://osu.ppy.sh/users/4956097), ::{ flag=MY }:: [\[Zeth\]](https://osu.ppy.sh/users/9912966) |
-| Screener | *TBD* |
+| Screener | ::{ flag=IT }:: [Arach](https://osu.ppy.sh/users/8512755), ::{ flag=AR }:: [Axer](https://osu.ppy.sh/users/7299864), ::{ flag=TR }:: [Burak](https://osu.ppy.sh/users/13213075), ::{ flag=HK }:: [Cynplytholowazy](https://osu.ppy.sh/users/3901754), ::{ flag=PH }:: [Eyenine](https://osu.ppy.sh/users/1259391), ::{ flag=FI }:: [Jaltzu](https://osu.ppy.sh/users/2597417), ::{ flag=US }:: [meiqth](https://osu.ppy.sh/users/12565402) |
 
 ## Prizes
 
-This contest features the re-introduction of *MBC points*, in the form of [contest points](/wiki/Contests/Contest_points), as a reward present in future official contests in order to provide a more streamlined path to the *Elite Mapper* title. The title is attainable by reaching 6 points.
+This contest featured the re-introduction of *MBC points*, in the form of [contest points](/wiki/Contests/Contest_points), as a reward present in future official contests in order to provide a more streamlined path to the *Elite Mapper* title. The title is attainable by reaching 6 points.
 
 | Placing | Prize(s) |
 | :-: | :-- |
 | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter, Ranked status[^prizes-ranked], unique profile badge, 3 points |
-| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter, Ranked status[^prizes-ranked], 2 points |
-| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu!supporter, Ranked status[^prizes-ranked], 1 point |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter, Ranked status[^prizes-ranked], unique profile badge, 2 points |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu!supporter, Ranked status[^prizes-ranked], unique profile badge, 1 point |
 
-![o!tfac #3 badge](img/badge.png "osu!taiko Featured Artist Cup #3 winning team")
+![o!tfac #3 badges](img/badge.png)
```

Comment:
can you put these as three images, like every other tournament/contest? [example](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/OWC/2022/en.md?plain=1#L37)